count/include/count_min_impl.hpp

```diff
@@ -0,0 +1,241 @@
+#ifndef COUNT_MIN_IMPL_HPP_
+#define COUNT_MIN_IMPL_HPP_
+
+#include "MurmurHash3.h"
+#include <random>
+
+namespace datasketches {
+
+template<typename W>
+count_min_sketch<W>::count_min_sketch(uint16_t num_hashes, uint32_t num_buckets, uint64_t seed):
+num_hashes(num_hashes),
+num_buckets(num_buckets),
+seed(seed){
+  total_weight = 0 ;
+  sketch_length = num_hashes*num_buckets ;
```

Comment:
this member variable does not seem to be necessary. it is easily derived and used once